packages/@aws-cdk/aws-elasticloadbalancingv2/lib/alb/application-listener.ts

```diff
@@ -186,6 +186,10 @@ export class ApplicationListener extends BaseListener implements IApplicationLis
     if (props.open !== false) {
       this.connections.allowDefaultPortFrom(ec2.Peer.anyIpv4(), `Allow from anyone on port ${port}`);
     }
+
+    if (props.open !== false && this.loadBalancer.ipAddressType === IpAddressType.DUAL_STACK) {
+      this.connections.allowDefaultPortFrom(ec2.Peer.anyIpv6(), `Allow from anyone on port ${port}`);
+    }
```

Comment:
why not just nest the `if (this.loadBalancer.ipAddressType === ...` condition into the if statement above?

it helps with readibility to keep the related logic in the same control flow block and saves on checking props.open a second time